airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/DisableActivityImpl.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.scheduling.activities;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.airbyte.config.JobConfig.ConfigType;
+import io.airbyte.config.StandardSync;
+import io.airbyte.config.StandardSync.Status;
+import io.airbyte.config.persistence.ConfigRepository;
+import io.airbyte.scheduler.models.Job;
+import io.airbyte.scheduler.models.JobStatus;
+import io.airbyte.scheduler.persistence.JobPersistence;
+import io.airbyte.workers.temporal.exception.RetryableException;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@AllArgsConstructor
+@Slf4j
+public class DisableActivityImpl implements DisableActivity {
+
+  @VisibleForTesting
+  public static final int MAX_FAILURE_JOBS_IN_A_ROW = 100;
```

Comment:
This should be coming from the `.env` file, which means that we need to update the Configs interface to add getter for those variables